### ISSUE TITLE
How should we design tests?

### ISSUE THREAD

**Issue by charleskawczynski** [climate-machine/CLIMAParameters.jl#7]
Designing tests for this repo is a bit tricky since all of the functions simply return values. So far, the incorporated tests are:

 - [x] Make sure the functions do not fail and do not return NaNs
 - [x] Test any functions that return relations (call other functions) to maintain consistency between the sets of hard-coded values

Are there any other tests that we incorporate? We could test hard-coded values, but they would be redundant in the `src/`. The only benefit I see here is that if a user changes a returned function value, they'll have to do it twice: once in the `src/` and once in the `test/`. This is arguably a way to make sure that developers are careful when changing values, but I'm not sure how useful that is.

**Comment by tapios** [comment]
I agree that testing hard-coded values is not useful. (And I would remove things like the test for length of (anomalistic) year that tests that this is 325.26*days. There are situations where we may want to change this and do not want to change test and src.) Testing the few thermodynamic relations you have is good. Otherwise, type stability and non-failure are the only crucial pieces. 

In other words: what you have is fine. I wouldn't spend much time on adding more here. Tests in other places of the code are more important.
